README.md

```diff
@@ -433,6 +433,24 @@ settings:
   import/internal-regex: ^@scope/
 ```
 
+#### `import/pnp`
```

Comment:
I don't think it's appropriate to add an option to the core rule for everyone, just for one package manager's nonstandard resolution.

This should be done via a separate package, like https://www.npmjs.com/package/eslint-import-resolver-node, that's specific to yarn pnp.